core/ledger/kvledger/txmgmt/statedb/commontests/test_common.go

```diff
@@ -1163,12 +1164,7 @@ func CreateTestData(t *testing.T, db statedb.VersionedDB, ns string, numKeys int
 type stringset []string
 
 func (s stringset) contains(str string) bool {
-	for _, element := range s {
-		if element == str {
-			return true
-		}
-	}
-	return false
+	return slices.Contains(s, str)
 }
```

Comment:
it is necessary not to change the body of the contains function, but the calls themselves. the contains function is no longer needed.